ios/Podfile

```diff
@@ -2,87 +2,17 @@
-  versions['Flipper-PeerTalk'] ||= '~> 0.0.4'
-  versions['Flipper-RSocket'] ||= '~> 1.1'
-
-  pod 'FlipperKit', versions['Flipper'], :configurations => configurations
-  pod 'FlipperKit/FlipperKitLayoutPlugin', versions['Flipper'], :configurations => configurations
-  pod 'FlipperKit/SKIOSNetworkPlugin', versions['Flipper'], :configurations => configurations
-  pod 'FlipperKit/FlipperKitUserDefaultsPlugin', versions['Flipper'], :configurations => configurations
-  pod 'FlipperKit/FlipperKitReactPlugin', versions['Flipper'], :configurations => configurations
-
-  # List all transitive dependencies for FlipperKit pods
-  # to avoid them being linked in Release builds
-  pod 'Flipper', versions['Flipper'], :configurations => configurations
-  pod 'Flipper-DoubleConversion', versions['Flipper-DoubleConversion'], :configurations => configurations
-  pod 'Flipper-Folly', versions['Flipper-Folly'], :configurations => configurations
-  pod 'Flipper-Glog', versions['Flipper-Glog'], :configurations => configurations
-  pod 'Flipper-PeerTalk', versions['Flipper-PeerTalk'], :configurations => configurations
-  pod 'Flipper-RSocket', versions['Flipper-RSocket'], :configurations => configurations
-  pod 'FlipperKit/Core', versions['Flipper'], :configurations => configurations
-  pod 'FlipperKit/CppBridge', versions['Flipper'], :configurations => configurations
-  pod 'FlipperKit/FBCxxFollyDynamicConvert', versions['Flipper'], :configurations => configurations
-  pod 'FlipperKit/FBDefines', versions['Flipper'], :configurations => configurations
-  pod 'FlipperKit/FKPortForwarding', versions['Flipper'], :configurations => configurations
-  pod 'FlipperKit/FlipperKitHighlightOverlay', versions['Flipper'], :configurations => configurations
-  pod 'FlipperKit/FlipperKitLayoutTextSearchable', versions['Flipper'], :configurations => configurations
-  pod 'FlipperKit/FlipperKitNetworkPlugin', versions['Flipper'], :configurations => configurations
-end
-
-# Post Install processing for Flipper
-def flipper_post_install(installer)
-  installer.pods_project.targets.each do |target|
-    if target.name == 'YogaKit'
-      target.build_configurations.each do |config|
-        config.build_settings['SWIFT_VERSION'] = '4.1'
-      end
-    end
-  end
-end
-
 target 'ZulipMobile' do
   # Pods from React Native
-  pod 'FBLazyVector', :path => "../node_modules/react-native/Libraries/FBLazyVector"
-  pod 'FBReactNativeSpec', :path => "../node_modules/react-native/Libraries/FBReactNativeSpec"
-  pod 'RCTRequired', :path => "../node_modules/react-native/Libraries/RCTRequired"
-  pod 'RCTTypeSafety', :path => "../node_modules/react-native/Libraries/TypeSafety"
+  use_react_native!
 
-  pod 'React', :path => '../node_modules/react-native/'
-  pod 'React-Core', :path => '../node_modules/react-native/'
-  pod 'React-CoreModules', :path => '../node_modules/react-native/React/CoreModules'
-  pod 'React-Core/DevSupport', :path => '../node_modules/react-native/'
```

Comment:
One change here is that this particular pod no longer gets included. The upstream function has this:
```ruby
  unless production
    pod 'React-Core/DevSupport', :path => "#{prefix}/"
  end
```

and `production` defaults to `false`.

I'm not sure what that does. It sounds like something that's good to leave out in release builds, but we might miss in dev builds. But I guess if things still seem to work well in development, then we didn't need it?

Reply:
Since `production` defaults to `false`, and we don't take it away from the default (which we would do by passing an argument with our `use_react_native!` call in the Podfile), I think this pod is still included in all builds.

```
  unless false
    // This code will definitely run
  end
```

Reply:
I've just changed it to `unless true` in node_modules and run `pod install` (via `yarn`), and I see a line with `React-Core/DevSupport` removed from `Podfile.lock`.

(The `/` says to look for `DevSupport` as a "subspec" of `React-Core`. I think all subspecs of `React-Core` would normally have been pulled in by the `pod 'React-Core'` line, above this conditional—but this isn't happening here because `React-Core`'s podspec declares a "default subspec"; [doc](https://guides.cocoapods.org/syntax/podspec.html#subspec).)

Probably best to keep pulling in `DevSupport` for development, but I agree that it should be fine to exclude it in production builds. I've just tried doing a release build with it excluded, and it built with no errors and is so far running just fine on my iPhone 6s.

I'd be nice if React Native took the approach it took with `use_flipper!` in facebook/react-native@c72ecdb906f29fbf9f97a2de2e9997a0f41d0352 (see also our 61bda2a20dfce2a21e3a485e73b43cbfe71a3985) toward `use_react_native!`. The "build configuration" (the thing we use to say whether we're building for production or development) isn't known at `pod install` time, which is the time the Podfile is used. So I don't see a way we could pass a correct value for the `production` param that `use_react_native!` expects.

Reply:
> So I don't see a way we could pass a correct value for the `production` param that `use_react_native!` expects.

Given this, it's possibly just fine to not do anything for now. Including `React-Core/DevSupport` in release builds is the status quo anyway.

Reply:
> `unless false` is equivalent to `if true`, I think. Is that right?

Ha, yes, I guess I misread that code :upside_down_face:

Reply:
Yeah, I agree with all the above. This change is NFC -- we were always getting this pod, and we're still always getting it. And the appropriate thing would be for that function to do something like what `use_flipper!` does, and take a list of configurations for it to apply in.

(Or else have some other way of being conditioned on the configuration.)